docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -27,10 +27,10 @@ Content-types in Strapi can be created:
 - with the [Content-Type Builder in the admin panel](/user-docs/latest/content-types-builder/introduction-to-content-types-builder.md),
 - or with [Strapi's interactive CLI `strapi generate`](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-generate) command.
 
-Creating a content-type with either method generates 2 files:
+The content-types has the following models files:
 
-- `schema.json` for the model's [schema](#model-schema) definition,
-- `lifecycles.js` for [lifecycle hooks](#lifecycle-hooks).
+- `schema.json` for the model's [schema](#model-schema) definition. (generated automatically, when creating content-type with either method)
```

Comment:
I don't think we need the () text at the end

Reply:
```suggestion
- `schema.json` for the model's [schema](#model-schema) definition. This file is automatically generated when creating a content-type.
```